openassessment/xblock/static/js/src/lms/oa_grade.js

```diff
@@ -20,17 +22,31 @@ OpenAssessment.GradeView.prototype = {
         var view = this;
         var baseView = this.baseView;
         var stepID = ".step--grade";
+        view.isRendering = true;
         this.server.render('grade').done(
             function(html) {
                 // Load the HTML and install event handlers
                 $(stepID, view.element).replaceWith(html);
                 view.server.renderLatex($(stepID, view.element));
+                view.isRendering = false;
                 view.installHandlers();
                 if (typeof usageID !== 'undefined' &&
                     !$(stepID, view.element).hasClass("is--unfinished") &&
                     !$(stepID, view.element).hasClass("is--unstarted") &&
                     !$(stepID, view.element).hasClass("is--waiting--staff")) {
                     $("[id='oa_grade_" + usageID + "']", view.element).focus();
+                    view.announceStatusChange = true;
+                }
+                if (typeof usageID !== 'undefined' &&
+                    $(stepID, view.element).hasClass("is--showing")){
+                    view.announceStatusChange = true;
```

Comment:
Can you explain why in this case we do want to announce a status change, but we don't want to focus (a comment in the code would be helpful)?